### PR TITLE
[DLS-8340] Fixing Sites CYA content

### PR DESCRIPTION
**Pull request by JakeReid2020**

### CODE REVIEW

File: app/controllers/CheckYourAnswersController.scala [hunk 1]
```diff
@@ -54,6 +54,8 @@ class CheckYourAnswersController @Inject()(
         returnsOrchestrator.calculateAmounts(sdilRef, userAnswers, returnPeriod).map { amounts =>
           val submitUrl: Call = routes.CheckYourAnswersController.onSubmit
 
+          println(s" USER ANSWERS = $userAnswers")
```
Comment (drewmiley) [inline app/controllers/CheckYourAnswersController.scala]: As platops says!

File: app/views/helpers/returnDetailsSummary.scala.html [hunk 2]
```diff
@@ -53,12 +53,25 @@ <h2 class="govuk-heading-m" id="lostOrDestroyed">@messages("lostOrDestroyed")</h
 <h2 class="govuk-heading-m" id="amount-to-pay-title">@AmountToPaySummary.sectionHeaderTitle(amounts.total)</h2>
 @govukSummaryList(AmountToPaySummary.amountToPaySummary(amounts))
 
+@if(userAnswers.packagingSiteList.nonEmpty || userAnswers.warehouseList.nonEmpty){
+ <h2 class="govuk-heading-m" id="registeredUkSites">@messages("registeredUkSites")</h2>
```
Comment (drewmiley) [inline app/views/helpers/returnDetailsSummary.scala.html:60]: Should we have 2 h2 headings in here?

File: app/viewmodels/checkAnswers/SecondaryWarehouseDetailsSummary.scala [hunk 3]
```diff
@@ -29,6 +29,33 @@ import viewmodels.implicits._
 
 object SecondaryWarehouseDetailsSummary  {
 
+  def summaryList(userAnswers: UserAnswers, isCheckAnswers: Boolean)
```
Comment (drewmiley) [inline app/viewmodels/checkAnswers/SecondaryWarehouseDetailsSummary.scala]: Is this used? I can't seem to see where it is called in the code?

File: app/views/helpers/returnDetailsSummary.scala.html (hunk 2 above)
Comment (JakeReid2020) [reply app/views/helpers/returnDetailsSummary.scala.html:60]: they show the same message for if either is true

Comment (drewmiley) [reply app/views/helpers/returnDetailsSummary.scala.html:60]: I mean could the h2 on line 57 and line 71 both show, and would that be correct?

File: app/views/helpers/returnDetailsSummary.scala.html [hunk 4]
```diff
@@ -53,12 +53,25 @@ <h2 class="govuk-heading-m" id="lostOrDestroyed">@messages("lostOrDestroyed")</h
...
  }
+}
+
+@AskSecondaryWarehouseSummary.summaryList(userAnswers, isCheckAnswers) match {
+ case Some(warehouse) => {
+  <h2 class="govuk-heading-m" id="registeredWarehouseSites">@messages("registeredUkSites")</h2>
+  @govukSummaryList(warehouse)
+ }
+ case _ => {
+  <br>
```
Comment (drewmiley) [inline app/views/helpers/returnDetailsSummary.scala.html:77]: Should we have a < br > if it's at the end of the file?

**Review by drewmiley — DISMISSED** [review]

**Review by drewmiley — APPROVED** [review]

**Pull request merged**